.github/workflows/ci.yaml

```diff
@@ -38,4 +38,56 @@ jobs:
         with:
           provider: lxd
       - name: Run integration tests
-        run: tox -e integration
+        run: tox -e integration-charm
+
+  integration-provider:
+    name: Integration tests for provider (lxd)
+    runs-on: ubuntu-latest
```

Comment:
As we're editing this, could we add a `needs` block to all these int-tests based on Jon's suggestion in MM?

```yaml
  integration-provider:
    name: Integration tests for provider (lxd)
    runs-on: ubuntu-latest
    needs:
      - lint
      - unit-test
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Setup operator environment
        uses: charmed-kubernetes/actions-operator@main
        with:
          provider: lxd
      - name: Run integration tests
        run: tox -e integration-provider
```

Reply:
I didn't catch the message. Is this to avoid running a bunch of time consuming tests when lint/unit doesn't pass?